sale_comment_template/tests/test_sale_order_report.py

```diff
@@ -2,9 +2,12 @@
 # Copyright 2018 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+import odoo.tests
 from odoo.tests.common import TransactionCase
 
 
+@odoo.tests.common.at_install(False)
```

Comment:
Why this decorator?

Reply:
All accounting related tests need to be tested at post install step.
The Chart of Account installation is done by the account module
'post_init_hook': '_auto_install_l10n'.
When testing the module at install, the configuration is not yet executed!

Otherwise following error is raised:
2019-03-25 09:17:54,765 32386 ERROR sale_comment_template odoo.addons.sale_comment_template.tests.test_sale_order_report: ` odoo.exceptions.UserError: ('Please define an accounting sales journal for this company.', '')

Reply:
Oh, weird...

Reply:
I think this functionality is changed somehow in Odoo 12.0 ...
All the accounting related standard Odoo tests are also marked as I did here.